go/libraries/doltcore/sqle/reflog_table_function.go

```diff
@@ -172,14 +166,11 @@ func (rltf *ReflogTableFunction) Schema() sql.Schema {
 }
 
 func (rltf *ReflogTableFunction) Resolved() bool {
-	if rltf.refExpr != nil {
-		return rltf.refExpr.Resolved()
-	}
 	return true
 }
 
 func (rltf *ReflogTableFunction) String() string {
-	return fmt.Sprintf("DOLT_REFLOG(%s)", rltf.refExpr.String())
+	return fmt.Sprintf("DOLT_REFLOG(%s)", rltf.refName)
```

Comment:
We want the result of this function to be a string representation of what query the user requested, but because we've already evaled the expression, this will show up as the result, not the actual expression. For example, if I call `select * from dolt_reflog(concat('h', 'e', 'll', 'o'));` then this will return `DOLT_REFLOG('hello')`, which loses some of the information about what query is being run. 

This function also needs to represent if the `--all` flag has been specified as a parameter.